### ISSUE TITLE
The keyboard `Enter` key fails to open the three dot action menu for email aliases on brave://settings/email-aliases page

### ISSUE THREAD

**Issue by GeetaSarvadnya** [brave/brave-browser#57938]
### Description

Found while testing https://github.com/brave/brave-browser/issues/57345

The keyboard `Enter` key fails to open the three dot action menu for email aliases on brave://settings/email-aliases page

### Steps to reproduce

1. Clean profile `1.94.104`
2. Create a Brave account 
3. Add a email aliases via brave://settings/email-aliases
4. Hit keyboard `TAB` key to navigate through 3 dot menu in Email aliases section 
5. Hit keyboard `ENTER` key on 3 dot menu
6. Notice that 3 dot menu isn't opened and Edit/Delete button is not shown 

### Actual result

The keyboard `Enter` key fails to open the three dot action menu for email aliases on brave://settings/email-aliases page

https://github.com/user-attachments/assets/4f29bb6a-2f69-4ebf-9c73-321da990a725

### Expected result

The keyboard `Enter` key should open the three dot action menu for email aliases on brave://settings/email-aliases page

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

Brave | 1.94.104 Chromium: 151.0.7922.83 (Official Build) beta (arm64)
-- | --
Revision | bdc8119dd5d54c7d2dbcb8f22223787686ac254c
OS | macOS Version 15.7.7 (Build 24G720)

### Channel information

- [ ] release (stable)
- [x] beta
- [ ] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

### Miscellaneous information

cc: @brave/qa-team @rebron @szilardszaloki

**Comment by szilardszaloki** [comment]
cc @boocmp

**Comment by LaurenWags** [comment]
cc @ShivanKaul for visibility
